src/v/rpc/transport.h

```diff
@@ -127,13 +127,14 @@ ss::future<result<rpc::client_context<T>>> parse_result(
         return parse_type<T>(in, sctx->get_header())
           .then_wrapped([sctx = std::move(sctx)](ss::future<T> data_fut) {
               if (data_fut.failed()) {
-                  sctx->body_parse_exception(data_fut.get_exception());
+                  auto ex = data_fut.get_exception();
```

Comment:
const auto?  - any of those methods take a mutable ref?

Reply:
fixed